remo/events/api/views.py

```diff
@@ -43,6 +43,13 @@ def retrieve(self, request, pk):
                                              context={'request': request})
         return Response(serializer.data)
 
+    def get_queryset(self):
+        orderby = self.request.query_params.get('orderby', 'DESC')
+
+        if orderby != 'DESC':
```

Comment:
This check is the wrong way around. We want the latest events shown on the first page by default (and also when using DESC actually). So this should be ```orderby  == 'DESC'```. See https://github.com/mozilla/remo/pull/1407 (which originally broke Community Analytics in the first place). For Community Analytics they will need to use the ```orderby=ASC``` filter in their configuration.

Default & DESC:
* Event 2018-02-28
* Event 2018-01-15
* Event 2017-11-15

ASC:
* Event 2017-11-15
* Event 2018-01-15
* Event 2018-02-28

Would you mind changing this?

Reply:
Uhm you are right is like the opposite right now.
I confused the order of the demo events.